ReactNative/ReactTwitter/core/views/main-navigator.js

```diff
@@ -19,11 +23,76 @@ const OauthViewID = 'oauth-screen-identifier'
 const tweetsListID = 'tweets-list-identifier'
 const tweetViewID = 'tweet-view-identifier'
 
-var MainNavigator = React.createClass({
+var NavigationBarRouteMapper = {
+  LeftButton (route, navigator, index, navState) {
+    if (index === 0) {
```

Comment:
It's the index of the current element in the navigation stack. 

So if we have alread pushed 3 scenes, we expect the navigation stack to contains these 3 references and the index to be 2